Tests/AK/TestTime.cpp

```diff
@@ -274,3 +274,130 @@ TEST_CASE(is_negative)
     EXPECT_EQ(result.to_nanoseconds(), 5);
     EXPECT(!result.is_negative());
 }
+
+struct YearAndDays {
+    int year;
+    int days;
+};
+
+TEST_CASE(years_to_days_since_epoch_points)
+{
+    Array<YearAndDays, 15> test_data = { {
+        { 1969, -365 },
+        { 1970, 0 },
+        { 1971, 365 },
+        { 1900, -25567 },
+        { 2023, 19358 },
+        { 1800, -62091 },
+        { 2100, 47482 },
+        { 0, -719528 },
+        { 4000, 741442 },
+        { -10000, -4371953 },
+        { 10000, 2932897 },
+        { -1000000, -365962028 },
+        { 1000000, 364522972 },
+        { -5877640, -2147483456 },
+        { 5881474, 2147444740 },
+    } };
+    for (auto entry : test_data) {
+        int year = entry.year;
+        int expected_days = entry.days;
+        int actual_days = years_to_days_since_epoch(year);
+        EXPECT_EQ(actual_days, expected_days);
+    }
+}
+
+TEST_CASE(years_to_days_since_epoch_benchmark)
```

Comment:
If this is meant to only be a benchmark, we have `BENCHMARK_CASE` for this.

Reply:
Good idea! I forgot that it exists, thanks!